src/rust/madara/sources/demonsect/res/source.json

```
@@ -0,0 +1,21 @@
+{
+	"info": {
+		"id": "pt-br.demonsect",
+		"lang": "pt-br",
+		"name": "Demon Sect",
+		"version": 1,
+		"url": "https://demonsect.com.br",
+		"nsfw": 2
+	},
+	"listings": [
+		{
+			"name": "Latest"
+		},
+		{
+			"name": "Trending"
+		},
+		{
+			"name": "Popular"
+		}
+	]
+}
```

Make sure there are newlines at the end of files
```suggestion
}

```